test/platform.ts

```diff
@@ -62,6 +67,11 @@ export interface IEmulatorManager {
      * Prepares the emulator for a test.
      */
     prepareEmulatorForTest(appId: string): Q.Promise<string>;
```

Comment:
but isn't the emulator manager the best place for simulator/emulator related operations